src/brasil/gov/tiles/tests/test_basic.py

```diff
@@ -33,7 +33,7 @@ def test_default_configuration(self):
 
     def test_accepted_content_types(self):
         registry = getUtility(IRegistry)
-        settings = registry.forInterface(ICoverSettings)
+        settings = registry.forInterface(ICoverSettings)  # noqa
```

Comment:
ao invés de fazer isso pode ignorar o código que gera o plugin no code analysis, assim fica um só lugar para mexer quando isso for consertado.